weather_mv/loader_pipeline/bq.py

```diff
@@ -230,8 +261,12 @@ def extract_rows(self, uri: str, coordinates: t.List[t.Dict]) -> t.Iterator[t.Di
                 row[DATA_IMPORT_TIME_COLUMN] = self.import_time
                 row[DATA_URI_COLUMN] = uri
                 row[DATA_FIRST_STEP] = first_time_step
-                row[GEO_POINT_COLUMN] = fetch_geo_point(row['latitude'], row['longitude'])
 
+                longitude = ((row['longitude'] + 180) % 360) - 180
+                row[GEO_POINT_COLUMN] = fetch_geo_point(row['latitude'], longitude)
+                row[GEO_POLYGON_COLUMN] = fetch_geo_polygon(row['latitude'], longitude,
+                                                            self.lat_grid_resolution, self.lon_grid_resolution
+                                                            ) if not self.skip_creating_polygon else None
```

Comment:
I find this a bit hard to read. Would you mind making it two lines?

Reply:
Code updated as per your suggestion.